pages/entropy/create-your-first-entropy-app.mdx

```diff
@@ -217,13 +229,12 @@ async main() {
 
   // Generate user random number and commitment
   const userRandomNumber = web3.utils.randomHex(32);
-  const userCommitment = web3.utils.keccak256(userRandomNumber);
 
   const fee = await entropyContract.methods.getFee(process.env["PROVIDER_ADDRESS"]).call()
   console.log(`fee         : ${fee}`);
 
   const requestReceipt = await coinFlipContract.methods
-    .request(userCommitment)
+    .request(userRandomNumber)
```

Comment:
you need to update the paragraph below this (it currently talks about hashing the user random number)